src-opam/dockerfile_distro.ml

```diff
@@ -217,7 +217,7 @@ let builtin_ocaml_of_distro (d:t) : string option =
   |`OpenSUSE `V15_2 -> Some "4.05.0"
   |`OracleLinux `V7 -> Some "4.01.0"
   |`OracleLinux `V8 -> Some "4.07.0"
-  |`Cygwin `V20H2 -> Some "4.10.0"
+  |`Cygwin `V21H1 -> Some "4.10.0"
```

Comment:
I'd change this to `None` on the basis that the OCaml in Cygwin64 at present is broken!